src/parser/ts-patch.js

```diff
@@ -28,6 +28,10 @@ try {
       readFile(fname) {
         let fileName = fname;
         let content = '';
+        if (fileName.endsWith('tsconfig.tsbuildinfo')) {
+          return content;
```

Comment:
Why?

Reply:
I don't know why tsconfig.tsbuildinfo was being passed in to here